app/cni/cmd/install/transform_test.go

```diff
@@ -0,0 +1,75 @@
+package main
+
+import (
+	"io/ioutil"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+const kumaCniConfig = `{
+		  "type": "kuma-cni",
+		  "log_level": "info",
+		  "kubernetes": {
+			"kubeconfig": "/etc/cni/net.d/ZZZ-kuma-cni-kubeconfig",
+			"cni_bin_dir": "/opt/cni/bin",
+			"exclude_namespaces": [
+			  "kuma-system"
+			]
+		  }
+		}`
+
+var _ = Describe("testTransformJsonConfig", func() {
+	It("should properly manipulate CNI conflist file", func() {
+		// given
+		calicoConfig, _ := ioutil.ReadFile("testdata/10-calico.conflist")
+		expectedConfig, _ := ioutil.ReadFile("testdata/10-calico.conflist.golden")
+
+		// when
+		result, err := transformJsonConfig(kumaCniConfig, calicoConfig)
+
+		// then
+		Expect(err).To(Not(HaveOccurred()))
+		Expect(result).To(MatchJSON(expectedConfig))
```

Comment:
consider using `MatchGoldenJson`. This way you can quickly update golden files with `UPDATE_GOLDEN_FILES=true`. Same for other tests